### PR TITLE
Using Common Db for cluster

### PR DESCRIPTION
**Pull request by Shifna12Zarnaz**

### CODE REVIEW

File: apps/conf/credentials/cosign-cred.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: cosign-cred-test
```
Comment (vramk23) [inline apps/conf/credentials/cosign-cred.yaml:1]: why "test" added with all names ?

it should be same as hard-coded names which removed

File: apps/conf/credentials/cosign-cred.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+secretName: cosign-keys-test
+namespaces:
+  - kyverno
+  - tekton-pipelines
+  - tek
+credentialEntity: cosign-test
+credentialIdentifier: signer-test
```
Comment (vramk23) [inline apps/conf/credentials/cosign-cred.yaml:8]: add credential-type, value will be like "cosign", "randomkey"

File: pkg/agent/store_cred.go [hunk 3]
```diff
@@ -178,16 +170,22 @@ func storeTerraformStateConfig(captenConfig config.CaptenConfig, agentClient age
 func storeNatsCredentials(captenConfig config.CaptenConfig, appGlobalVaules map[string]interface{}, vaultClient vaultcredpb.VaultCredClient) error {
 	val, err := randomTokenGeneration()
 	if err != nil {
-		return fmt.Errorf("Nats Token generation failed, %v", err)
+		return fmt.Errorf("nats Token generation failed, %v", err)
+	}
+
```
Comment (vramk23) [inline pkg/agent/store_cred.go:175]: it need to read all yaml files, based on yaml credential-type and other values, it need to create credential and update in capten

**Review by vramk23 — CHANGES_REQUESTED** [review]

File: apps/conf/credentials/cosign-cred.yaml (hunk 2 above)
Comment (Shifna12Zarnaz) [reply apps/conf/credentials/cosign-cred.yaml:8]: instead of credential entity field,Should I need to use credential-type

**Review by venkatk63 — APPROVED** [review]

**Pull request merged**